ksqldb-api-client/pom.xml

```diff
@@ -95,6 +95,16 @@
             <version>${reactive-streams.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-annotations</artifactId>
+        </dependency>
+
```

Comment:
The diff does not show that we "remove" the "skip config" for spotbugs, because it chancels out with the change from #7808 that are included in this PR. After #7808 is merged and this PR is rebased, the change should show up on Github diff.